contracts/components/staking/SubjectTypeValidator.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-network/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.9;
+
+uint8 constant UNDEFINED_SUBJECT = 255;
+uint8 constant SCANNER_SUBJECT = 0;
+uint8 constant AGENT_SUBJECT = 1;
+uint8 constant NODE_RUNNER_SUBJECT = 2;
+uint8 constant DELEGATOR_NODE_RUNNER_SUBJECT = 3;
```

Comment:
I'm confused by this distinction. What is a "delegator node runner"?

And it seems we changed the value of `NODE_RUNNER_SUBJECT`, it used to be 3 and is now 2. Is this ok?